### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by nileshtrivedi** [edgedb/edgedb#7359]
Both the `curl > sh` and `npx` method are failing for me. The following gets stuck on "downloading installer" and never proceeds...
```
% curl --proto '=https' --tlsv1.2 -sSf https://sh.edgedb.com | sh
info: downloading installer
```

Whereas I get a Timeout error with `npx`:

```
% npx edgedb
No EdgeDB CLI found, downloading CLI package...
node:internal/deps/undici/undici:12344
    Error.captureStackTrace(err, this);
          ^

TypeError: fetch failed
    at node:internal/deps/undici/undici:12344:11
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async getVersionMap (file:///Users/mac/code/learn-anything.xyz/node_modules/edgedb/dist/cli.mjs:219:26)
    at async findPackage (file:///Users/mac/code/learn-anything.xyz/node_modules/edgedb/dist/cli.mjs:209:24)
    at async downloadCliPackage (file:///Users/mac/code/learn-anything.xyz/node_modules/edgedb/dist/cli.mjs:182:20)
    at async getCliLocationFromTempCli (file:///Users/mac/code/learn-anything.xyz/node_modules/edgedb/dist/cli.mjs:148:5)
    at async main (file:///Users/mac/code/learn-anything.xyz/node_modules/edgedb/dist/cli.mjs:42:10)
    at async file:///Users/mac/code/learn-anything.xyz/node_modules/edgedb/dist/cli.mjs:26:1 {
  cause: ConnectTimeoutError: Connect Timeout Error
      at onConnectTimeout (node:internal/deps/undici/undici:7492:28)
      at node:internal/deps/undici/undici:7448:50
      at Immediate._onImmediate (node:internal/deps/undici/undici:7480:13)
      at process.processImmediate (node:internal/timers:478:21) {
    code: 'UND_ERR_CONNECT_TIMEOUT'
  }
}

Node.js v20.11.0
```

Because neither of these scripts log what they are trying to download and where from, I am unable to confirm if something is broken with my DNS/Internet/SSL etc.

**Comment by raddevon** [comment]
The installer is downloaded from `https://packages.edgedb.com/<architecture>/edgedb-cli<extension>`.

`<extension>` is `.exe` if you're on Windows or nothing elsewhere.
For `<architecture>`, the script runs `uname -s` and `uname -m` to determine the CPU type and OS, normalizes them, and combines them in the format `<cpu-type>-<os-type>`.

It looks like you're on a Mac, so your URL would be either `https://packages.edgedb.com/dist/aarch64-apple-darwin/edgedb-cli` if you have an Apple Silicon Mac or `https://packages.edgedb.com/dist/x86_64-apple-darwin/edgedb-cli` if you have an Intel Mac.

You might check your Mac's network settings and see if the firewall is on. If so, try disabling it and see if that helps.

**Comment by scotttrinh** [comment]
You can also get more debug information from the `npx` version by setting `DEBUG=*` on the command line like:

```
DEBUG=* npx edgedb
```

Feel free to share your results here, or if the output is sensitive, feel free to email me at scott@edgedb.com.

**Comment by nileshtrivedi** [comment]
The `curl | sh` method worked fine when I ran this using a different Internet connection (home v/s work).

**Comment by scotttrinh** [comment]
The npx installation issue has been fixed in v1.5.7, thanks for the details!
